elastic/logs/challenges/many-shards-quantitative.json

```diff
@@ -84,5 +84,6 @@
       }
     },
     {% include "tasks/field-caps.json" %}
```

Comment:
You'll need a comma at the end of this line to maintain valid json

Reply:
Thanks! Fixed now :D